openapi/refs.go

```diff
@@ -8,15 +8,16 @@ import (
 
 // readRef takes a reference and attempts to unmarshal it's content into the struct being passed as `i`.
 // As it happens, this ref is contained within the struct that is being unmarshalled into.
+// TODO can `i` become a non-empty interface? perhaps [Node | Traversable]?
 func readRef(absFilename string, i interface{}) error {
-	fmt.Printf("Reading reference: %s\n", absFilename)
+	if Debug { // this can be a particularly noisy Printf call
```

Comment:
Consider adding a cli flag to set Debug to true or false. If you feel that's beyond the scope of this PR, would you consider creating an issue for it?

Reply:
Sure, I'll do that. 
Given the hope for adoption and quality of feedback we may receive through making that a public feature, I really like that idea